utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -1395,6 +1385,37 @@ Describe 'Module tests' -Tag 'Module' {
             }
         }
 
+        It '[<moduleFolderName>] A [` version.json `] file must only have a major & minor version.' -TestCases ($moduleFolderTestCases | Where-Object { $_.versionFileExists }) {
+
+            param (
+                [string] $moduleFolderPath
+            )
+
+            $versionFileContent = Get-Content (Join-Path -Path $moduleFolderPath 'version.json') | ConvertFrom-Json -AsHashtable
+            $versionFileContent.version | Should -Match '^[0-9]+\.[0-9]+$' -Because 'only the major.minor version may be specified in the version.json file.'
+        }
+
+        # Temporary test, before v1.0 release
+        It '[<moduleFolderName>] A [` version.json `] file must not yet contain a major version greater than 0.' -TestCases ($moduleFolderTestCases | Where-Object { $_.versionFileExists }) {
+
+            param (
+                [string] $moduleFolderPath,
+                [string] $moduleType,
+                [string] $moduleFolderName
+            )
+
+            if ($moduleType -eq 'res' -and $moduleFolderName -eq 'network/nat-gateway') {
+                # Using a warning and skip, since nat-gateway has already been released with version 1.x.
+                Write-Warning "[avm/$moduleType/$moduleFolderName] has already been released with version 1.x"
+                Set-ItResult -Skipped -Because 'the module has already been released with version 1.x.'
+                return
+            }
+
+            $versionFileContent = Get-Content (Join-Path -Path $moduleFolderPath 'version.json') | ConvertFrom-Json -AsHashtable
+            $major, $minor = $versionFileContent.version -split '\.'
+            $major | Should -Be 0 -Because 'module version must be incremented via minor and patch versions only for the time being.'
```

Comment:
```suggestion
            $major | Should -Be 0 -Because 'module version must be incremented via minor and patch (automatic) versions only for the time being.'
```